test/test_pyops.py

```diff
@@ -147,7 +147,7 @@ def join(xs, **kwargs):
             res = []
             for x in xs:
                 res.append(sep.join(x))
-            return np.array(res, dtype=np.object)
```

Comment:
according to the warning, 
DeprecationWarning: `np.object` is a deprecated alias for the builtin `object`. To silence this warning, use `object` by itself

should we use `object' or np.object_ here?

Reply:
thanks for the suggestion. updated np.object_ to object